pkg/jiraservicedesk/client/client_customer.go

```diff
@@ -0,0 +1,116 @@
+package client
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"strconv"
+
+	jiraservicedeskv1alpha1 "github.com/stakater/jira-service-desk-operator/api/v1alpha1"
+)
+
+const (
+	// Endpoints
+	EndpointCreateCustomer = "/rest/servicedeskapi/customer"
+	EndpointAddCustomer    = "/rest/servicedeskapi/servicedesk/"
```

Comment:
`/rest/servicedeskapi/servicedesk/` seems very generic. Are you sure it's the endpoint to add customers ?

Reply:
yep it is :p

Reply:
Well, it isn't :P because this particular slug represents their API path and then `/rest/servicedeskapi/servicedesk/{project-id}/customers` represents the endpoint to associate a customer to a project